### PR TITLE
Docs: style numbers (en).

### PR DESCRIPTION
**Pull request by afilahkle**
**Description**

This PR tries to unify the style of code numbers across the English documentation.

### CODE REVIEW

File: docs/api/en/math/Vector4.html [hunk 1]
```diff
@@ -325,13 +325,13 @@ <h3>
 			[method:this setComponent]( [param:Integer index], [param:Float value] )
 		</h3>
 		<p>
-			[page:Integer index] - 0, 1 or 2.<br />
```
Comment (afilahkle) [inline docs/api/en/math/Vector4.html:328]: Hi @Mugen87 ,

i think this line should be `[page:Integer index] - 0, 1, 2 or 3.<br />` instead.
if i'm right, should i file a PR for it?

Comment (Mugen87) [reply docs/api/en/math/Vector4.html:328]: You are right! Fixing this in a separate PR would be great 👍 .

**Review by Mugen87 — APPROVED** [review]

File: docs/api/en/math/Vector4.html (hunk 1 above)
Comment (afilahkle) [reply docs/api/en/math/Vector4.html:328]: cool, will do that

**Comment by mrdoob** [comment]
I'm not sure we want to do this. That style is meant to be used for cost values and not for any number.

**Comment by afilahkle** [comment]
Okay, make sense 
i'll try to do that, just to make sure i understand correctly,
by code we mean a number that is for example a Default value or a parameter that take values between say `x` and `y`.
right?

**Comment by Mugen87** [comment]
> by code we mean a number that is for example a Default value or a parameter that take values between say x and y.
right?

Yes, I would start just with default values. As an example, the `BoxGeometry` page could be updated from:

> defaults to 1

to 

> defaults to `1`

**Review by Mugen87 — APPROVED** [review]

**Pull request merged**